### PR TITLE
topydo: add French translation

### PR DESCRIPTION
**Pull request by melvinxeu**
- [X] The page (if new), does not already exist in the repository.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.fr/common/topydo.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# topydo
+
+> Une application de liste de choses à faire qui utilise le format todo.txt.
+> Plus d'informations: <https://github.com/topydo/topydo>.
```
Comment (navarroaxel) [inline pages.fr/common/topydo.md:4]: We use this French space:
```suggestion
> Plus d'informations : <https://github.com/topydo/topydo>.
```

File: pages.fr/common/topydo.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Ajouter une tâche à un projet spécifique avec un contexte donné:
```
Comment (navarroaxel) [inline pages.fr/common/topydo.md:6]: I think this is the correct form of the présent de l'indicatif
```suggestion
- Ajoute une tâche à un projet spécifique avec un contexte donné:
```

**Comment by navarroaxel** [comment]
@melvinxeu could you please apply my suggestions?

**Comment by navarroaxel** [comment]
@nicokosi @patricedenis could you please help here with the verbs?

File: pages.fr/common/topydo.md (hunk 2 above)
Comment (nicokosi) [reply pages.fr/common/topydo.md:6]: `ajouter` (infinitive) is OK, it follows the [style guide's "imperative mood"](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood).

File: pages.fr/common/topydo.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# topydo
+
+> Une application de liste de choses à faire qui utilise le format todo.txt.
+> Plus d'informations : <https://github.com/topydo/topydo>.
+
+- Ajouter une tâche à un projet spécifique avec un contexte donné:
```
Comment (nicokosi) [inline pages.fr/common/topydo.md:6]:
```suggestion
- Ajouter une tâche à un projet spécifique avec un contexte donné :
```

File: pages.fr/common/topydo.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`topydo add "{{todo_message}} +{{projet_nom}} @{{context_nom}}"`
+
+- Ajouter une tâche à faire avec une date d'échéance de demain et une priorité de `A`:
```
Comment (nicokosi) [inline pages.fr/common/topydo.md:10]:
```suggestion
- Ajouter une tâche à faire avec une date d'échéance de demain et une priorité de `A` :
```

File: pages.fr/common/topydo.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`topydo add "(A) {{todo _message}} due:{{1d}}"`
+
+- Ajoutez une tâche à faire dont la date d'échéance est le vendredi:
```
Comment (nicokosi) [inline pages.fr/common/topydo.md:14]:
```suggestion
- Ajouter une tâche à faire dont la date d'échéance est le vendredi :
```

File: pages.fr/common/topydo.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+`topydo add "{{todo_message}} due:{{fri}}"`
+
+- Ajouter une tâche répétitive non stricte (prochain due = maintenant + reçu):
```
Comment (nicokosi) [inline pages.fr/common/topydo.md:18]:
```suggestion
- Ajouter une tâche répétitive non stricte (prochaine échéance = maintenant + récurrence) :
```

File: pages.fr/common/topydo.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+`topydo add "water flowers due:{{mon}} rec:{{1w}}"`
+
+- Ajouter une tâche répétitive stricte (prochaine échéance = échéance actuelle + reçu):
```
Comment (nicokosi) [inline pages.fr/common/topydo.md:22]:
```suggestion
- Ajouter une tâche répétitive stricte (prochaine échéance = maintenant + récurrence) :
```

File: pages.fr/common/topydo.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+`topydo add "{{todo_message}} due:{{2020-01-01}} rec:{{+1m}}"`
+
+- Revenir sur la dernière commande `topydo` exécutée:
```
Comment (navarroaxel) [inline pages.fr/common/topydo.md:26]:
```suggestion
- Revenir sur la dernière commande `topydo` exécutée :
```

**Review by navarroaxel — APPROVED** [review]

**Review by nicokosi — APPROVED** [review]
Well done, thanks ! :+1: (_bien joué, merci !_ :stuck_out_tongue_winking_eye:)

**Pull request merged**